composer.json

```diff
@@ -17,10 +16,11 @@
         }
     ],
     "require": {
-        "cakephp/cakephp": "^3.5"
+        "cakephp/cakephp": "^3.7"
```

Comment:
Why is bumping CakePHP version needed?

Reply:
I have reverted back the Cake version bump. The couple of method exists check in helpers aren't really a problem.